packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/documents/DocumentControllerIntegrationTests.java

```diff
@@ -125,8 +125,8 @@ public void givenStudioWhenTheCreationOfDomainDocumentIsRequestedThenTheDomainIs
 
     @Test
     @GivenSiriusWebServer
-    @DisplayName("Given a studio, when the upload of a new domain document is performed, then the domain is created")
-    public void givenStudioWhenTheUploadOfDomainDocumentIsPerformedThenTheDomainIsCreated() {
+    @DisplayName("Given a studio, when the upload of a new domain document with read-write permission is performed, then the domain is created")
```

Comment:
I'll revert most changes to existing test to remove this notion of permission. We do not support permissions.